### ISSUE TITLE
TV mode

### ISSUE THREAD

**Issue by DumberSoft** [ValerioLyndon/Rockfin#3]
Also check if it is compatible with TV mode or if it comes by default like Jellyfin.

**Comment by ValerioLyndon** [comment]
TV Mode? You mean like Roku TVs and stuff or is there a separate TV mode? I'll have to look into it once desktop & mobile are up to a standard I find acceptable.

**Comment by DumberSoft** [comment]
Of course, like Roku TVs, or it can be Samsung TV or LG Webos, however, the profile also has the configuration to put TV mode and you can see the designs... To do this, go to configuration and there you will find it as TV Mode.
